src/patternfly/components/Login/login.scss

```diff
@@ -17,21 +17,33 @@
 
   // header
   --pf-c-login__header--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-login__header--sm--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-login__header--sm--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-login__header--PaddingLeft: var(--pf-global--spacer--md);
```

Comment:
Just noting, this is an example of 6 lines of code turning into 12. There are a couple of those in this component.